pages/team.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: page
+title: MDAnalysis team
+---
+
+MDAnalysis is a community-driven project that is made possible through the efforts of many members who contribute in numerous and diverse ways, ranging from direct package development, maintenance, documentation, communication, and managerial responsibilities. On this page we list identified project roles and those historically responsible for the tasks. In the future, we plan to also list the contributors currently responsible for overseeing and managing associated tasks. We note that the listed roles on this page can differ significantly in scope and required effort.
+
```

Comment:
The community is somewhat small and I think there is good historical knowledge. But maybe worth adding something on the lines of "if you have participated to MDAnalysis in the past and feel your previous/historical role could be better characterised, please contact us at [...]"?

Reply:
Thanks for this note @RMeli, I've used mdanalysis@numfocus.org for now but that may be too broad of a destination.